everything_else/reproducing.m

```diff
@@ -349,6 +349,14 @@
 %% Pg. 55, SCM-BCR
 %% Pg. 56, Decomposition into symmetric and anti-symmetric parts
 
+b = dec2bin(2^4-1:-1:0)-'0';
+b1 = b(:,1); b2 = b(:,2); b3 = b(:,3); b4 = b(:,4);
+
+f=b1.*b2 + b2.*b3 + b3.*b4 - 4*b1.*b2.*b3;
+
```

Comment:
It would be better if this example had the following:
 - 1 linear term with a coefficient of +1, 
 - 1 quadratic terms with a coefficient of +2, 
 - 1 cubic term with a coefficient of -4, and 
 - one quartic term with a coefficient of +2.

Reply:
I will add that asap.